packages/helper-grammar-regex-collection/test.js

```diff
@@ -400,6 +400,18 @@ const fixtures = {
       'import com.company..myapp',
     ],
   },
+  NET_PROJ_PACKAGE: {
+    valid: [
+      '<PackageReference Include="EntityFramework" Version="6.2.0" />',
+      '< PackageReference Include="EntityFramework" Version="6.2.0" />',
```

Comment:
Are you sure that a space in front of `PackageReference ` is a valid format?

Reply:
Did you run `yarn test` locally? I think your tests will fail. By default the test suite which is made out of this config expects "foo" as target.

You could either change the lines to

```js
<PackageReference Include="foo" Version="6.2.0" />
```
or use an array where the second item descibes the expected result.

```js
['<PackageReference Include="EntityFramework" Version="6.2.0" />','EntityFramework']
```

Reply:
As far as I know a space beween `<` and the tag name isn't valid, but I wasn't sure if github may add space when rendering the html (the `NET_PACKAGE` regex had it, so I followed how that was setup). I've removed it though since it doesn't seem right and we can always adjust it later if there's situations where it may be needed.